draft-jhoyla-req-mtls-flag.md

```diff
@@ -133,7 +133,81 @@ TLS namespace with the following values:
 
 --- back
 
+# Appendix
+
+## Alternative Approaches
+
+### HTTP Signatures
+
+HTTP Signatures {{?I-D.draft-meunier-web-bot-auth-architecture}} describes a
+mechanism for implementing bearer tokens at the HTTP layer that authenticate
+requests. This mechanism suffers from a number of drawbacks, notably that a new
+signature must be created and verified for every request (rather than one per
+connection), and second that, because the tokens are not bound to the underlying
+TLS connection they can be stolen and replayed. This is especially problematic
+in the multi-CDN use case. If a bot creates a token for a realm that is provided
+by more than one CDN then the CDN that receives that token can use it to attack
+any of the other CDNs. This means that all CDNs are required to trust each
+other.
+
+### Concealed Auth
+
+Concealed Auth {{?RFC9729}} allows a client to attach a header at the HTTP
+layer that cryptographically binds a public key to the underlying TLS
+connection. This gives us a similar authentication property to using
+`request_client_auth`, and is also client initiated. This solution is
+applicable in some contexts, but in the case of authenticating bots may be
+difficult to implement.
```

Comment:
I don't see how the next paragraph is about bots specifically: see  but in the case of authenticating bots may be
difficult to implement.

```suggestion
applicable in some contexts, but may be
difficult to implement.
```